submodules.json

```diff
@@ -17,9 +26,14 @@
   {
     "name": "jquery-boilerplate",
     "url": "https://github.com/jquery-boilerplate/jquery-boilerplate",
-    "tags": ["jquery", "jqueryjs", "js"]
+    "tags": ["jquery", "jqueryjs", "js"],
+    "owner": "jquery-boilerplate"
   },
   {
+    "name": "android-boilerplate",
+    "url": "https://github.com/ribot/android-boilerplate",
+    "tags": ["android", "mvvm", "rxjava"],
+    "owner": "ribot"
     "name": "nginx-boilerplate",
     "url": "https://github.com/nginx-boilerplate/nginx-boilerplate",
```

Comment:
There are duplicate values and I don't think you need to add a separate field for owner as you can just parse it from the URL. For example, if the URL is this - https://github.com/IEEE-VIT/templa-rs the repo owner can be parsed from this string itself.
You can just take 1 argument and take the last part of the URL i.e., `IEEE-VIT/templa-rs` and query the API with that